### PR TITLE
LPD-15718 Add a new SEO menu in Instance Settings to include specific Sites in the sitemap of Company

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-15718

### CODE REVIEW

File: modules/apps/site/site-api/src/main/java/com/liferay/site/configuration/manager/SitemapConfigurationManager.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
  */
 public interface SitemapConfigurationManager {
```
Comment (ealonso) [inline modules/apps/site/site-api/src/main/java/com/liferay/site/configuration/manager/SitemapConfigurationManager.java:13]: @lfbesada if you add @ProviderType it is not going to be necessary a breaking change report

Comment (lfbesada) [reply modules/apps/site/site-api/src/main/java/com/liferay/site/configuration/manager/SitemapConfigurationManager.java:16]: @ealonso Thanks for the tip :star_struck:  I've added it to the PR

**Comment by jkappler** [comment]
looks good @lfbesada ❤️. Just the comments, but we cannot change it......so from my side, all okay with the backend side

**Comment by lfbesada** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/15158

**Pull request closed without merging**